### ISSUE TITLE
Error on login, and Appearances tab still not skinned with ElvUI

### ISSUE THREAD

**Issue by antedap** [SLOKnightfall/BetterWardrobe#301]
1x Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:122: attempt to index local 'WardrobeCollectionFrame' (a nil value)
[string "@Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua"]:122: in function <Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:118>
[string "@Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua"]:466: in function <Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:464>

Locals:
WardrobeCollectionFrame = nil
(*temporary) = <function> defined =[C]:-1
(*temporary) = <table> {
 L_UIDropDownMenuSkinned = true
 Blizzard_Channels = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Channels.lua:7
 TalkingHead = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/TalkingHead.lua:110
 HandleFrame = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:129
 ArrowRotation = <table> {
 }
 HandleNextPrevButton = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1089
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceTimer-3.0/AceTimer-3.0.lua:147
 QuestFrameProgressItems_Update = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Quest.lua:337
 GuildInviteFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Guild.lua:217
 Blizzard_ItemSocketingUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Socket.lua:10
 Ace3_ColorizeEnable = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Ace3.lua:512
 AlertSystem = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Alerts.lua:787
 HandleIcon = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1027
 Blizzard_WeeklyRewards = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/WeeklyRewards.lua:65
 SetDisabledBackdrop = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:234
 HandleCategoriesButtons = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:76
 Blizzard_PlayerChoice = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/PlayerChoice.lua:107
 ADDON_LOADED = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1783
 HandleSliderFrame = <function> defined =[C]:-1
 SkinTalentListButtons = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:360
 HandleEditBox = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:829
 TutorialFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/TutorialFrame.lua:6
 db = <table> {
 }
 SetEnabledState = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceHook-3.0/AceHook-3.0.lua:273
 RegisterSkin = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1818
 DressUpFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/DressingRoom.lua:49
 Blizzard_AnimaDiversionUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/AnimaDiversion.lua:6
 HandleFollowerListOnUpdateDataFunc = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1397
 SkinDoubleStateIconRow = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1737
 HandleTrimScrollBar = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:669
 SkinWidgetContainer = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1776
 HandleCheckBox = <function> defined =[C]:-1
 UnregisterAllEvents = <function> defined @Interface/AddOns/AllTheThings/lib/CallbackHandler-1.0/CallbackHandler-1.0.lua:174
 Blizzard_ItemInteractionUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/ItemInteraction.lua:6
 ScriptErrorsFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Debug.lua:86
 Blizzard_Calendar = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Calendar.lua:34
 SkinCaptureZone = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1746
 EarlyAceWidgets = <table> {
 }
 Blizzard_DebugTools = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Debug.lua:92
 QuestFrame_SetTitleTextColor = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Quest.lua:342
 name = "ElvUI_Skins"
 Blizzard_CovenantSanctum = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/CovenantSanctum.lua:50

![Screenshot_41](https://user-images.githubusercontent.com/73392980/200521710-cda849f2-3cdd-4a17-a649-99b2ae50d66b.png)

**Comment by Beet4** [comment]
Can confirm. Still an issue with latest version 4.3.

```
1x Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:122: attempt to index local 'WardrobeCollectionFrame' (a nil value)
[string "@Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua"]:122: in function <Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:118>
[string "@Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua"]:466: in function <Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:464>

Locals:
WardrobeCollectionFrame = nil
(*temporary) = <function> defined =[C]:-1
(*temporary) = <table> {
 L_UIDropDownMenuSkinned = true
 Blizzard_Channels = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Channels.lua:7
 TalkingHead = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/TalkingHead.lua:110
 HandleFrame = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:129
 ArrowRotation = <table> {
 }
 HandleNextPrevButton = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1089
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceTimer-3.0/AceTimer-3.0.lua:147
 QuestFrameProgressItems_Update = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Quest.lua:337
 GuildInviteFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Guild.lua:217
 Blizzard_ItemSocketingUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Socket.lua:10
 Ace3_ColorizeEnable = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Ace3.lua:512
 AlertSystem = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Alerts.lua:787
 HandleIcon = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1027
 Blizzard_WeeklyRewards = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/WeeklyRewards.lua:65
 SetDisabledBackdrop = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:234
 HandleCategoriesButtons = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:76
 Blizzard_PlayerChoice = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/PlayerChoice.lua:107
 ADDON_LOADED = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1783
 HandleSliderFrame = <function> defined =[C]:-1
 SkinTalentListButtons = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:360
 HandleEditBox = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:829
 TutorialFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/TutorialFrame.lua:6
 db = <table> {
 }
 SetEnabledState = <function> defined @Interface/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceHook-3.0/AceHook-3.0.lua:273
 RegisterSkin = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1818
 DressUpFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/DressingRoom.lua:49
 Blizzard_AnimaDiversionUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/AnimaDiversion.lua:6
 HandleFollowerListOnUpdateDataFunc = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1397
 SkinDoubleStateIconRow = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1737
 HandleTrimScrollBar = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:669
 SkinWidgetContainer = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1776
 HandleCheckBox = <function> defined =[C]:-1
 UnregisterAllEvents = <function> defined @Interface/AddOns/Masque/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:169
 Blizzard_ItemInteractionUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/ItemInteraction.lua:6
 ScriptErrorsFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Debug.lua:86
 Blizzard_Calendar = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Calendar.lua:34
 SkinCaptureZone = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1746
 EarlyAceWidgets = <table> {
 }
 Blizzard_DebugTools = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Debug.lua:92
 QuestFrame_SetTitleTextColor = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Quest.lua:342
 name = "ElvUI_Skins"
 Blizzard_CovenantSanctum = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/CovenantSanctum.lua:50
 SkinIconTextAndC
```

![image](https://user-images.githubusercontent.com/38279393/200672856-d1d56811-0e43-4a09-9571-94d378333536.png)

**Comment by dewson8** [comment]
having the same issue even with the new 4.3.2.

**Comment by matttb** [comment]
Having same issue 

`1x Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:122: attempt to index local 'WardrobeCollectionFrame' (a nil value)
[string "@Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua"]:122: in function <Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:118>
[string "@Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua"]:468: in function <Interface/AddOns/BetterWardrobe/Plugins/ElvUI.lua:464>

Locals:
WardrobeCollectionFrame = nil
(*temporary) = <function> defined =[C]:-1
(*temporary) = <table> {
 Blizzard_Channels = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Channels.lua:7
 TalkingHead = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/TalkingHead.lua:110
 HandleFrame = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:129
 ArrowRotation = <table> {
 }
 HandleNextPrevButton = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1084
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceTimer-3.0/AceTimer-3.0.lua:147
 QuestFrameProgressItems_Update = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Quest.lua:335
 GuildInviteFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Guild.lua:217
 Blizzard_ItemSocketingUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Socket.lua:10
 Ace3_ColorizeEnable = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Ace3.lua:512
 AlertSystem = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Alerts.lua:787
 HandleIcon = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1022
 Blizzard_WeeklyRewards = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/WeeklyRewards.lua:65
 SetDisabledBackdrop = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:234
 HandleCategoriesButtons = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:76
 Blizzard_PlayerChoice = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/PlayerChoice.lua:107
 ADDON_LOADED = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1778
 HandleSliderFrame = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1162
 SkinTalentListButtons = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:360
 HandleEditBox = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:824
 TutorialFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/TutorialFrame.lua:6
 db = <table> {
 }
 SetEnabledState = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @Interface/AddOns/ElvUI/Libraries/Core/Ace/AceHook-3.0/AceHook-3.0.lua:273
 RegisterSkin = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1813
 DressUpFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/DressingRoom.lua:49
 Blizzard_AnimaDiversionUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/AnimaDiversion.lua:6
 HandleFollowerListOnUpdateDataFunc = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1392
 SkinDoubleStateIconRow = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1732
 HandleTrimScrollBar = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:664
 SkinWidgetContainer = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1771
 HandleCheckBox = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:905
 UnregisterAllEvents = <function> defined @Interface/AddOns/AskMrRobot/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:200
 Blizzard_ItemInteractionUI = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/ItemInteraction.lua:6
 Blizzard_ProfessionsCustomerOrders = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/ProfessionsCustomerOrders.lua:28
 ScriptErrorsFrame = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Debug.lua:86
 Blizzard_Calendar = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Calendar.lua:34
 SkinCaptureZone = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua:1741
 EarlyAceWidgets = <table> {
 }
 Blizzard_DebugTools = <function> defined @Interface/AddOns/ElvUI/Mainline/Modules/Skins/Debug.lua:92
 QuestFrame_SetTitleTextColor = <function> defi`

**Comment by SLOKnightfall** [comment]
Should be fixed in 4.3.2
